src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -50,6 +50,9 @@ internal class StartHostAction : BaseAction
         private const string InProc6DirectoryName = "in-proc6";
         private const string InProc8HostRuntime = "inproc8";
         private const string InProc6HostRuntime = "inproc6";
+        private const string InProcFunctionsSdk = "Microsoft.NET.Sdk.Functions";
```

Comment:
Commented below, but this information shouldn't be required.

Reply:
Using sdk name just to print in the log message. Let me know if any correction is needed in the log.